homeassistant/components/qbittorrent/sensor.py

```diff
@@ -95,7 +144,56 @@ class QBittorrentSensorEntityDescription(SensorEntityDescription):
         native_unit_of_measurement=UnitOfDataRate.BYTES_PER_SECOND,
         suggested_display_precision=2,
         suggested_unit_of_measurement=UnitOfDataRate.MEGABYTES_PER_SECOND,
-        value_fn=get_up,
+        value_fn=get_upload_speed,
+    ),
+    QBittorrentSensorEntityDescription(
+        key=SENSOR_TYPE_DOWNLOAD_SPEED_LIMIT,
+        translation_key="download_speed_limit",
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.DATA_RATE,
+        native_unit_of_measurement=UnitOfDataRate.BYTES_PER_SECOND,
+        suggested_display_precision=2,
+        suggested_unit_of_measurement=UnitOfDataRate.MEGABYTES_PER_SECOND,
+        value_fn=get_download_speed_limit,
+        entity_registry_enabled_default=False,
+    ),
+    QBittorrentSensorEntityDescription(
+        key=SENSOR_TYPE_UPLOAD_SPEED_LIMIT,
+        translation_key="upload_speed_limit",
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.DATA_RATE,
+        native_unit_of_measurement=UnitOfDataRate.BYTES_PER_SECOND,
+        suggested_display_precision=2,
+        suggested_unit_of_measurement=UnitOfDataRate.MEGABYTES_PER_SECOND,
+        value_fn=get_upload_speed_limit,
+        entity_registry_enabled_default=False,
+    ),
+    QBittorrentSensorEntityDescription(
+        key=SENSOR_TYPE_ALLTIME_DOWNLOAD,
+        translation_key="alltime_download",
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        device_class=SensorDeviceClass.DATA_SIZE,
+        native_unit_of_measurement="B",
+        suggested_display_precision=2,
+        suggested_unit_of_measurement="TiB",
```

Comment:
```suggestion
        suggested_unit_of_measurement=UnitOfInformation.TEBIBYTES,
```
same for the other one